src/platform/intel/cavs/platform.c

```diff
@@ -173,8 +174,7 @@ struct work_queue_timesource platform_generic_queue[] = {
 	.timer_clear	= platform_timer_clear,
 	.timer_get	= platform_timer_get,
 },
-#if defined(CONFIG_CANNONLAKE) || defined(CONFIG_ICELAKE) \
-	|| defined(CONFIG_SUECREEK)
+#if CAVS_VERSION >= CAVS_VERSION_1_8
```

Comment:
Seems to be independent change, good candidate for a dedicated commit.

Reply:
I've moved CAVS_VERSION defines change to another commit